Frontend/src/test/components/ui/toaster.test.jsx

```
@@ -1,32 +1,55 @@
+  ToastClose: vi.fn(),
+  ToastViewport: vi.fn(),
 }))
 
-import { Toaster } from '../../../components/ui/toaster'
-
 describe('Toaster component', () => {
+  beforeEach(() => {
+    // Default toasts
+    vi.mocked(useToast).mockReturnValue({
+      toasts: [
+        { id: '1', title: 'Title 1', description: 'Desc 1', action: <button data-testid="action-1">A1</button> },
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)